packages/terra-date-time-picker/src/_TimeClarification.module.scss

```diff
@@ -1,29 +1,29 @@
 /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
 :local {
   .time-clarification {
```

Comment:
nit: I'd drop the word modal from these variables and go with something like: `--terra-date-time-picker-time-clarification-background-color`. The thought being if we every switch time verification from a clarification, (which we probably won't) we could keep using this variable name.